packages/kokkos-kernels/src/sparse/impl/KokkosSparse_spgemm_impl_kkmem.hpp

```diff
@@ -40,8 +40,11 @@
 // ************************************************************************
 //@HEADER
 */
+
 #define HASHSCALAR 107
```

Comment:
We need to fix that macro sometime.  Macros that are exposed in header files must have package namespacing.